src/ios/BranchSDK.m

```diff
@@ -379,91 +339,67 @@ - (void)createBranchUniversalObject:(CDVInvokedUrlCommand*)command
     NSLog(@"start createBranchUniversalObject");
 
     NSDictionary *properties = [command.arguments objectAtIndex:0];
-    self.branchUniversalObj = [[BranchUniversalObject alloc] init];
+    BranchUniversalObject *branchUniversalObj = [[BranchUniversalObject alloc] init];
 
     for (id key in properties) {
         if ([key isEqualToString:@"contentMetadata"]){
             NSDictionary *metadata = (NSDictionary *)[properties valueForKey:key];
 
             for (id key_ in metadata) {
-                [self.branchUniversalObj addMetadataKey:key_ value:[metadata valueForKey:key_]];
+                [branchUniversalObj addMetadataKey:key_ value:[metadata valueForKey:key_]];
             }
         }
         else if ([key isEqualToString:@"contentIndexingMode"]) {
             NSString *indexingMode = [properties valueForKey:key];
             if ([indexingMode isEqualToString:@"private"]) {
-                self.branchUniversalObj.contentIndexMode = ContentIndexModePrivate;
+                branchUniversalObj.contentIndexMode = ContentIndexModePrivate;
 
             }
             else if ([indexingMode isEqualToString:@"public"]){
-                self.branchUniversalObj.contentIndexMode = ContentIndexModePublic;
+                branchUniversalObj.contentIndexMode = ContentIndexModePublic;
             }
```

Comment:
Should there be a default we want to specify here, or just deeper in the SDK? If we want it deeper (so we don't duplicate default logic) a comment here would be great.